ResearchKit/Common/ORKHealthAnswerFormat.m

```diff
@@ -35,9 +35,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import "ORKDefines_Private.h"
 
 
-#if !defined(__IPHONE_8_2)
 #define HKBiologicalSexOther 3
```

Comment:
Almost! Remove the HKBiologicalSexOther redefinition too, that was only for pre-8.2 tools.